src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/multinomial.cpp

```diff
@@ -0,0 +1,142 @@
+                                          10.0f,
+                                          0.1f,
+                                          0.1f,
+                                          10.0f,
+                                          0.00001f,
+                                          0.001f,
+                                          10.0f,
+                                          0.1f,
+                                          0.001f,
+                                          0.00001f};
+
+const std::vector<ov::float16> probs_2x3_f16 = {(ov::float16)0.001f,
+                                                (ov::float16)0.1f,
+                                                (ov::float16)10.0f,
+                                                (ov::float16)10.0f,
+                                                (ov::float16)0.001f,
+                                                (ov::float16)0.1f};
+
+const std::vector<ov::bfloat16> probs_1x3_bf16 = {(ov::bfloat16)0.001f, (ov::bfloat16)0.1f, (ov::bfloat16)10.0f};
+
+const std::vector<float> probs_4x4_f32_log =
+    {3.0f, 6.0f, 10.0f, 0.0f, 3.0f, 0.0f, 10.0f, 6.0f, 6.0f, 10.0f, 0.0f, 3.0f, 10.0f, 6.0f, 3.0f, 0.0f};
+
+const std::vector<ov::float16> probs_2x3_f16_log = {(ov::float16)3.0f,
+                                                    (ov::float16)6.0f,
+                                                    (ov::float16)10.0f,
+                                                    (ov::float16)10.0f,
+                                                    (ov::float16)3.0f,
+                                                    (ov::float16)6.0f};
+
+const std::vector<ov::bfloat16> probs_1x3_bf16_log = {(ov::bfloat16)3.0f, (ov::bfloat16)6.0f, (ov::bfloat16)10.0f};
+
+const std::vector<int> num_samples_scalar_i32 = {1};
+const std::vector<int64_t> num_samples_1x1_i64 = {2};
+const std::vector<int64_t> num_samples_scalar_i64 = {3};
+
+const std::vector<ov::Tensor> probs = {ov::Tensor(ov::element::f32, {4, 4}, (void*)probs_4x4_f32.data()),
+                                       ov::Tensor(ov::element::f16, {2, 3}, (void*)probs_2x3_f16.data()),
+                                       ov::Tensor(ov::element::bf16, {1, 3}, (void*)probs_1x3_bf16.data())};
+
+const std::vector<ov::Tensor> probs_log = {ov::Tensor(ov::element::f32, {4, 4}, (void*)probs_4x4_f32_log.data()),
+                                           ov::Tensor(ov::element::f16, {2, 3}, (void*)probs_2x3_f16_log.data()),
+                                           ov::Tensor(ov::element::bf16, {1, 3}, (void*)probs_1x3_bf16_log.data())};
+
+const std::vector<ov::Tensor> num_samples = {ov::Tensor(ov::element::i32, {}, (void*)num_samples_scalar_i32.data()),
+                                             ov::Tensor(ov::element::i64, {1}, (void*)num_samples_1x1_i64.data()),
+                                             ov::Tensor(ov::element::i64, {}, (void*)num_samples_scalar_i64.data())};
```

Comment:
To fix CI: 
[log](https://openvino-ci.toolbox.iotg.sclab.intel.com/blue/organizations/jenkins/private-ci%2Fie%2Fbuild-linux-macos/detail/build-linux-macos/197858/pipeline/)
```
multinomial.cpp:61:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead  
multinomial.cpp:62:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead  
multinomial.cpp:63:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead  
multinomial.cpp:65:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead  
multinomial.cpp:66:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead  
multinomial.cpp:67:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead  
multinomial.cpp:69:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead  
multinomial.cpp:70:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead  
multinomial.cpp:71:  Using C-style cast.  Use reinterpret_cast<void*>(...) instead
```

Reply:
Applied!

Reply:
Unfortunately it triggered another issue:
https://github.com/openvinotoolkit/openvino/actions/runs/6889989707/job/18742105998?pr=20406#step:13:37790
Maybe it's better to use a helper function to create those Tensors.

```
/intel_cpu/tests/functional/single_layer_tests/instances/x64/multinomial.cpp:61:77: error: reinterpret_cast from 'const float *' to 'void *' casts away qualifiers
const std::vector<ov::Tensor> probs = {ov::Tensor(ov::element::f32, {4, 4}, reinterpret_cast<void*>(probs_4x4_f32.data())),
```